woocommerce-gutenberg-products-block.php

```diff
@@ -6,8 +6,7 @@
  * Version: 1.1.2
  * Author: Automattic
  * Author URI: https://woocommerce.com
- * Text Domain:  woocommerce
- * Domain Path:  /languages
+ * Text Domain:  woo-gutenberg-products-block
```

Comment:
We can remove the "Domain Path" because we're listed in wordpress.org [source](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#domain-path)